### ISSUE TITLE
about specifying controls/backgrounds

### ISSUE THREAD

**Issue by tanpuekai** [kundajelab/atac_dnase_pipelines#87]
Hi, Jin,

thanks a lot for your previous help with the "six" package, that resolved the ATAQC errors.

another question (hopefully the last) i want to ask is, how to specify an input sample as controls (or background)?

I noted that one can specify "replicates" for both single-end and pair-end, in your pipeline. but cannot seem to find a way to specify "controls"?

for example, i may have 8 fastq files:

- signal_rep1_1.fastq
- signal_rep1_2.fastq
- signal_rep2_1.fastq
- signal_rep2_2.fastq
- control_rep1_1.fastq
- control_rep1_2.fastq
- control_rep2_1.fastq
- control_rep2_2.fastq

is there a way to run the pipeline with the "controls"?

best
Chan

**Comment by leepc12** [comment]
No, we are sorry that our ATAC-Seq pipeline does not support controls...
